src/cljs/rems/administration/status_flags.cljs

```diff
@@ -35,7 +35,7 @@
         label (if enabled?
                 :t.administration/disable
                 :t.administration/enable)]
-    {:id (or id "toggle-enabled")
+    {:id id
```

Comment:
Could add "toggle-enabled" as class as this would be consistent with the other buttons.